pkg-manager/resolve-dependencies/src/index.ts

```diff
@@ -304,6 +305,8 @@ export async function resolveDependencies (
     }
   }
 
+  newLockfile.catalogs = getCatalogSnapshots(Object.values(resolvedImporters).flatMap(x => x.directDependencies))
```

Comment:
Don't use meaningless variable names. Also, in this case you can avoid the variable
```suggestion
  newLockfile.catalogs = getCatalogSnapshots(Object.values(resolvedImporters).flatMap(({ directDependencies }) => directDependencies))
```

Reply:
Done!

I think using a short name when calling `.map` is common though. For example, in Swift, there's a `$0` shorthand for this built into the programming language. Usually the short name isn't meaningless since the plural name is right next to it — `resolvedImporters` in this case.

https://docs.swift.org/swift-book/documentation/the-swift-programming-language/closures/#Shorthand-Argument-Names